packages/DataHeader/stories/DataHeader.stories.mdx

```diff
@@ -0,0 +1,154 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { getStoryName } from "storybook/storyTree";
+import Readme from "../README.md";
+import { Basic, WithRenderActions, WithTypes, WithBackgroundColor } from "./DataHeader.examples";
+import Toast from "@paprika/toast";
+
+export const params = {
+  options: {
+    isToolshown: true,
+    showPanel: false,
+  },
+};
+
+<Meta title={`${getStoryName("DataHeader")}/Docs`} parameters={params} />
+
+# DataHeader
+
+## Introduction
+
+The `<DataHeader />` component helps consumer to deal with header with the `<Table />` and `<DataGrid />` components.
+
+With this component you have access to three major features: render Label via a string or function, render an action button/menu at the right side of the header or render an icon `type`
+before the label.
+
+The component looks like:
+
+```js
+<DataHeader label={string|function} type={:string} renderActions={:function} />
+```
+
+## Basic example
+
+A basic example using a `<Table />`
+
+```js
+<Table data={data}>
+  <Table.ColumnDefinition header={() => <DataHeader label="name" />} cell="name" />
+</Table>
+```
+
+### Output
+
+<Basic />
+
+## Render Actions Prop
+
+The previous example is too simple, you can achieve the same using a single `header` prop on the `ColumnDefinition`
+Let's add the `renderActions` prop to make it more interesting:
+
+```js
+import DropdownMenu from "@paprika/dropdown-menu";
+
+function Menu() {
+  return (
+    <DropdownMenu>
+      <DropdownMenu.Trigger buttonType="raw">
+        <EllipsisVertical />
+      </DropdownMenu.Trigger>
+      <DropdownMenu.Item onClick={() => {}}>One</DropdownMenu.Item>
+      <DropdownMenu.Item onClick={() => {}}>Two</DropdownMenu.Item>
+      <DropdownMenu.Item onClick={() => {}}>Three</DropdownMenu.Item>
+    </DropdownMenu>
+  );
+}
+
+function App() {
+  return (
+    <Table data={data}>
+      <Table.ColumnDefinition
+        width="150px"
+        header={() => <DataHeader label="name" renderActions={() => <Menu />} />}
+        cell="name"
+      />
+    </Table>
+  );
+}
+```
+
+### Output
+
+<WithRenderActions />
+<br /><br />
+<Toast hasCloseButton={false}>
+  You can render whatever you need into the `renderActions` space not only a menu, can be a button, icon, etc. It will always be align to the right.
+</Toast>
```

Comment:
Oh! I _like_ this 🤩 Nice easy way to make a callout.  
cc @oscarkwan @KaanDarcey 

However the `renderActions` just displays with backticks, you'd have to use a `<code>` block or something.